sass/themes/frost/components/_form.scss

```diff
@@ -205,3 +206,10 @@ form {
     @extend .link--blue;
   }
 }
+
+.form__overlay {
+  position: absolute;
+  background: rgba(255, 255, 255, 0.7);
+  width: 100%;
+  height: 100%;
+}
```

Comment:
can you add a new line here please? so github stops complaining and we keep this as a standard on our files.